core/src/main/scala/filodb.core/query/RangeVector.scala

```diff
@@ -127,17 +215,18 @@ final case class ChunkInfoRangeVector(key: RangeVectorKey,
 }
 
 /**
-  * SerializableRangeVector represents a RangeVector that can be serialized over the wire.
+  * SerializedRangeVector represents a RangeVector that can be serialized over the wire.
   * RecordContainers may be shared amongst all the SRV's from a single Result to minimize space and heap usage --
   *   this is the reason for the startRecordNo, the row # of the first container.
   * PLEASE PLEASE use Kryo to serialize this as it will make sure the single shared RecordContainer is
   * only serialized once as a single instance.
   */
-final class SerializableRangeVector(val key: RangeVectorKey,
-                                    val numRowsInt: Int,
-                                    containers: Seq[RecordContainer],
-                                    val schema: RecordSchema,
-                                    startRecordNo: Int) extends RangeVector with java.io.Serializable {
+final class SerializedRangeVector(val key: RangeVectorKey,
+                                  val numRowsInt: Int,
+                                  containers: Seq[RecordContainer],
+                                  val schema: RecordSchema,
+                                  startRecordNo: Int) extends RangeVector with SerializableRangeVector with
+  java.io.Serializable {
```

Comment:
better to put all the extends and with's to a single line together

Reply:
Also, java.io.Serializable should NOT be included.  We don't want to use Java serialization with this class, because it already serializes into a binary record.  Instead we just want Kryo serialization and to make sure Kryo serializer is used, in Serializer.scala.    If we need to include Serializable, then we are using the wrong path.  Please leave it out.

Reply:
I just renamed SerializableRangeVector to SerializedRangeVector and it had java.io.Serializable. We don't need it now? Just want to make sure that it won't break any existing code :)